spec/models/user_spec.rb

```diff
@@ -34,9 +34,21 @@
   end
 
   describe 'User' do
-    subject { create(:user, admin: false) }
```

Comment:
I think the `describe 'User' do` spec here was meant to describe a user without administration rights (as opposed to the spec above it (`describe 'Admin' do`). So that doesn't seem the right place for the full_name test.

In keeping with the conventions in the rest of the spec file I would expect a `describe #full_name do` spec for testing the full_name method.

Reply:
agree with @RobertCram

Reply:
Sounds good, I wasn't sure which of the two routes to go.